src/macro_util.rs

```diff
@@ -146,10 +146,16 @@ macro_rules! trailing_field_offset {
             }
         };
 
-        assert!(min_size <= _64K);
+        if min_size > $crate::macro_util::_64K {
+            panic!(concat!(
+                "cannot compute layout of `",
+                stringify!($ty),
+                "`; type is larger than zerocopy supports"
```

Comment:
Leaving a TODO so we don't forget: We should have an issue link here so there's a way for users to report that they need support for larger types.